### PR TITLE
FI-3357: Add Carin For Blue Button Resources

### PR DESCRIPTION
**Pull request by emichaud998**
# Summary
Add Bundle to `resources` directory that includes examples of resources for each Carin for Blue Button profile with each MS element populated.

## Testing guidance
Ensure Bundle of Carin resources is valid and can be accessed on the Inferno Reference server

### CODE REVIEW

**Review by karlnaden — APPROVED** [review]
reference server runs locally and loads the additional data. CARIN client tests run against it.

**Comment by dehall** [comment]
I ran this through the validator and got a few errors. If these don't affect the test kit then I won't worry about them too much but seems like they might be things you want to fix?

>   Error @ Bundle.entry[7].resource/*ExplanationOfBenefit/c4bb-EOBOral*/.item[0] (line 1599, col12): Slice 'ExplanationOfBenefit.item.adjudication:adjudicationamounttype': a matching slice is required, but not found (from http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-ExplanationOfBenefit-Oral|2.0.0). Note that other slices are allowed in addition to this required slice
>  Error @ Bundle.entry[6].resource/*ExplanationOfBenefit/c4bb-EOBOutpatient*/ (line 866, col8): Constraint failed: EOB-institutional-item-or-header-adjudication: 'Institutional EOB:  Should have adjudication with adjudicationamounttype slice at the item or header level, but not both' (defined in http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-ExplanationOfBenefit-Outpatient-Institutional)
>  Error @ Bundle.entry[9].resource/*ExplanationOfBenefit/c4bb-EOBProfessional*/ (line 2321, col8): Constraint failed: EOB-prof-all-transportation-supportinginfo-linked-to-line: 'Professional EOB: SupportingInfo repetitions with with transportation category code must be referred to by one or more repetitions of item.informationSequence' (defined in http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-ExplanationOfBenefit-Professional-NonClinician)

Unfortunately the file takes too long on validator.fhir.org so I had to run it with the validator jar, for reference:  `java -jar validator_cli_6_4_2.jar -version 4.0.1 -ig "hl7.fhir.us.carin-bb#2.1.0-snapshot1" carin_bundle_patient_888.json`

File: resources/carin_bundle_patient_888.json [hunk 1]
```diff
@@ -0,0 +1,2843 @@
...
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Coverage",
+        "id": "c4bb-Coverage",
+        "meta": {
+          "lastUpdated": "2020-10-30T09:48:01.8462752-04:00",
+          "profile": [
+            "http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-Coverage|2.1.0-snapshot1"
```
Comment (karlnaden) [inline resources/carin_bundle_patient_888.json:112]: version should be 2.0.0, not the snapshot

File: resources/carin_bundle_patient_888.json [hunk 2]
```diff
@@ -0,0 +1,2843 @@
...
+        "url": "Coverage/c4bb-Coverage"
+      }
+    },
+    {
+      "resource": {
+        "resourceType" : "Organization",
+        "id" : "c4bb-Organization",
+        "meta" : {
+          "lastUpdated" : "2020-05-04T03:02:01-04:00",
+          "profile" : ["http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-Organization|2.1.0-snapshot1"]
```
Comment (karlnaden) [inline resources/carin_bundle_patient_888.json:222]: version should be 2.0.0, not the snapshot

File: resources/carin_bundle_patient_888.json [hunk 3]
```diff
@@ -0,0 +1,2843 @@
...
+        "url": "Organization/c4bb-Organization"
+      }
+    },
+    {
+      "resource": {
+        "resourceType" : "Practitioner",
+        "id" : "c4bb-Practitioner",
+        "meta" : {
+          "lastUpdated" : "2020-05-04T03:02:01-04:00",
+          "profile" : ["http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-Practitioner|2.1.0-snapshot1"]
```
Comment (karlnaden) [inline resources/carin_bundle_patient_888.json:302]: version should be 2.0.0, not the snapshot

File: resources/carin_bundle_patient_888.json [hunk 4]
```diff
@@ -0,0 +1,2843 @@
...
+        "url": "Practitioner/c4bb-Practitioner"
+      }
+    },
+    {
+      "resource": {
+        "resourceType" : "RelatedPerson",
+        "id" : "c4bb-RelatedPerson",
+        "meta" : {
+          "lastUpdated" : "2020-05-04T03:02:01-04:00",
+          "profile" : ["http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-RelatedPerson|2.1.0-snapshot1"]
```
Comment (karlnaden) [inline resources/carin_bundle_patient_888.json:364]: version should be 2.0.0, not the snapshot

**Review by karlnaden — APPROVED** [review]
added resources now pass validation

**Pull request merged**